### DISCUSSION TITLE
what to do if your commits aren't showing on the website

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#37] [Admin General]
i want everybody to know this if they didn't already. the reason your commits aren't showing on the website is because browsers save javascript files and the like to avoid loading them every time a user clicks onto a new page or reloads. there's an easy fix to this though

**easy way:** ctrl+shift+r / ctrl+F5

**hard way:**

go to **chrome://settings** on your chrome browser, **edge://settings** on microsoft edge, or **about:config** on firefox
![Screenshot from 2024-03-18 15-49-55](https://github.com/mariocraft987/bark.github.io/assets/129226914/328e4e51-4b09-46e9-9bed-2850068b8933)
search for **"cache"** or go to **privacy & security**
![Screenshot from 2024-03-18 15-50-10](https://github.com/mariocraft987/bark.github.io/assets/129226914/36b6aff4-80d1-4f2d-9396-73114a251723)
click **"clear browsing data"** and uncheck everything except **"cached images and files"**
![Screenshot from 2024-03-18 15-50-20](https://github.com/mariocraft987/bark.github.io/assets/129226914/1c25e614-5bd6-4efb-9edf-97ca0df0ea56)
click **"clear data"**

**Comment by mariocraft987** [comment]
👍

**Reply by mariocraft987** [reply]
ctrl + f5 if you want another option
